openwisp_network_topology/static/netjsongraph/css/src/netjsongraph.css

```
@@ -164,27 +158,54 @@ body {
-
-.njg-date {
+.njg-container .njg-date {
   position: absolute;
   right: 20px;
   bottom: 20px;
   z-index: 1;
 }
 
-.njg-selectIcon {
+@font-face {
+  font-family: "iconfont";
```

Stylelint (17.14.0)

[error] Expected no quotes around "iconfont" (font-family-name-quotes)

(font-family-name-quotes)